src/platform/posix.c

```diff
@@ -479,23 +548,51 @@ void platform_enumerate_directories_end(int handle)
 }
 
 int platform_get_drives(){
-	/*
-	return GetLogicalDrives();
-	*/
-	STUB();
+	// POSIX systems do not know drives. Return 0.
 	return 0;
 }
 
 bool platform_file_copy(const utf8 *srcPath, const utf8 *dstPath, bool overwrite)
 {
-	STUB();
+	log_verbose("Copying %s to %s", srcPath, dstPath);
+
+	// If overwrite is not on and the file already exists, return 0
+	if(!overwrite && platform_file_exists(dstPath)) {
+		log_warning("platform_file_copy: Not overwriting %s, because overwrite flag == false", dstPath);
+		return 0;
+	}
+
+
+	// Open both files and check whether they are opened correctly
+	FILE *srcFile = fopen(srcPath, "r");
```

Comment:
`fopen` automatically converts newline chars for you, this is probably not what you want. To surpress this behaviour, pass the `b` flag as well. Like so, `fopen(srcPath, "rb")`.